mirgecom/initializers.py

```diff
@@ -0,0 +1,218 @@
+__copyright__ = """
```

Comment:
`mirgecom.initial_conditions`? `initials`? `ics`?

("Initializer" means something CS-y, which is a bit related, but to me not quite.)

Reply:
It initializes (in the CS sense) the solution.